.expeditor/bundle_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -evx
+
+gem environment
+bundler_version=$(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
```

Comment:
the appveyor config used `$env:BUNDLER_VERSION=` which actively failed on bundler 1.14.x and i changed that to `$env:OMNIBUS_BUNDER=` here:

https://github.com/chef/chef/commit/a3159a69242adc94050e1664a7c1ced9d08a84a6#diff-180360612c6b8c4ed830919bbb4dd459